### PR TITLE
en: add document about how to do performance test

### PR DESCRIPTION
**Pull request by toutdesuite**
### What is changed, added or deleted? (Required)

- add document about how to do the performance test
- update `benchmark-v1.0-ga.md`
- TOC updated in https://github.com/pingcap/docs-dm/pull/133
- merge https://github.com/pingcap/docs-dm/pull/129 then merge this PR

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v2.0 changes for now)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-dm/pull/85
- Other reference link(s):

### CODE REVIEW

File: en/performance-test.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+  target-database:
+    host: "192.168.0.1"
+    port: 4000
+    user: "root"
+    password: ""
+
+  mysql-instances:
+    -
+    source-id: "source-1"
+      block-allow-list:  "instance"
```
Comment (ran-huang) [inline en/performance-test.md:73]: @WangXiangUSTC The indentation is different from [the Chinese version](https://github.com/pingcap/docs-dm/pull/85/files#diff-d958db6377242948ab6dd57e69222859R72). Is this correct?

File: en/performance-test.md [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+ [INFO] [loader.go:604] ["all data files have been finished"] [task=test] [unit=load] ["cost time"=52.439796ms]
+```
+
+According to the size of the test data and the time consumed to import data, you can calculate the replication speed of the full data.
+
+### Incremental replication benchmark case
+
+#### Initialize tables
+
+Use `sysbench` to create test tables upstream.
```
Comment (ran-huang) [inline en/performance-test.md:109]:
```suggestion
Use `sysbench` to create test tables in the upstream.
```

File: en/performance-test.md [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+
+For details about how to create a data replication task, see [Create a Data Replication Task](create-task.md).
+
+> **Note:**
+>
+> To test the performance under different configurations, you can tune `worker-count` and `batch` in the `syncers` configuration item.
+
+#### Generate incremental data
+
+Run the `sysbench` command to continuously generate incremental data upstream:
```
Comment (ran-huang) [inline en/performance-test.md:155]:
```suggestion
To continuously generate incremental data in the upstream, run the `sysbench` command:
```

File: en/performance-test.md (hunk 1 above)
Comment (toutdesuite) [reply en/performance-test.md:73]: I‘ll update it

File: en/performance-test.md [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+
+## Deploy test environment
+
+- Deploy the TiDB test cluster using TiUP, with all default configurations.
+- Deploy the MySQL service. Enable the `ROW` mode for binlog, and use default configurations for other configuration items.
+- Deploy a DM cluster, with a DM-worker and a DM-master.
+
+## Performance test
+
+### Table structure
```
[the remark this answers is not in the document]
Comment (toutdesuite) [reply en/performance-test.md:22]: Confirmed. The second one is used more often

File: en/performance-test.md [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+    password: ""
+
+  mysql-instances:
+    -
+      source-id: "source-1"
+      block-allow-list:  "instance"
+      mydumper-config-name: "global"
+      loader-thread: 16
+
+  # Configure the name of the library where sysbench generates data.
```
Comment (WangXiangUSTC) [inline en/performance-test.md:76]: do we need to change `library` to `database`?

File: en/performance-test.md [hunk 6]
```diff
@@ -0,0 +1,168 @@
...
+    user: "root"
+    password: ""
+
+  mysql-instances:
+    -
+      source-id: "source-1"
+      block-allow-list:  "instance"
+      syncer-config-name: "global"
+
+  # Configure the name of the library where sysbench generates data.
```
Comment (WangXiangUSTC) [inline en/performance-test.md:135]: ditto

File: en/performance-test.md (hunk 5 above)
Comment (ran-huang) [reply en/performance-test.md:76]: It depends on the actual meaning of the Chinese version. Is it `library` or `database`?

Comment (WangXiangUSTC) [reply en/performance-test.md:76]: this means database name

Comment (ran-huang) [reply en/performance-test.md:76]: Sure. Let's use `database` @toutdesuite

Comment (toutdesuite) [reply en/performance-test.md]: updated in d0cddfb860cb7f76ba2e413025667f37e12e3bda~

**Pull request merged**